pages/common/glab-issue.md

```diff
@@ -0,0 +1,24 @@
+# glab issue
+
+> Manage Gitlab issues from the command-line.
+> More information: <https://glab.readthedocs.io/en/latest/issue/>.
+
+- Print out the issue:
```

Comment:
I prefer replace the phrasal verb to a simpler English verb:
 ```suggestion
- Display a specific issue:
```
Easier to read for basic English speakers.